ddebug: include c99_alloca.h for alloca() (illumos declares it in <alloca.h>).

Upstreamable.

--- src/gallium/auxiliary/driver_ddebug/dd_draw.c.orig
+++ src/gallium/auxiliary/driver_ddebug/dd_draw.c
@@ -26,6 +26,8 @@
  **************************************************************************/
 
 #include "dd_pipe.h"
+
+#include "c99_alloca.h"
 
 #include "util/os_file.h"
 #include "util/u_dump.h"
